Modules/FixDaDatabase.md

```diff
@@ -0,0 +1,16 @@
+# Name: FixDaDatabase
+
+## Description:
+This module has been created to remediate a common problem with Direct Access servers. It can determine if an Index is present for the Direct Access is present. And it can either remove and create that index.
```

Comment:
Please review this statement: "determine if an Index is present for the Direct Access is present." double "is present"? 😄